install/db/achecker_schema.sql

```diff
@@ -29,7 +29,7 @@ CREATE TABLE `checks` (
   `test_failed_result` text,
   `func` text,
   `open_to_public` tinyint(4) NOT NULL DEFAULT '0',
-  `create_date` datetime NOT NULL,
+  `create_date` datetime DEFAULT NULL,
```

Comment:
There should always be a `create_date` value. The original `datetime NOT NULL` should be correct.

Reply:
When updating it to "NOT NULL",  should I also update all the create_date columns in the check table to "0" instead of the previous NULL value

Reply:
that is to remove all current NULL values and replace it with something else

Reply:
Yes, you are right. It's a historical issue that `create_date` for system level checks were set to '0000-00-00 00:00:00'. Let's use your approach of `DEFAULT NULL`. Thanks for the discovery.

Reply:
Ok, thanks, will do that